src/app/garfile/view/get.controller.js

```diff
@@ -2,6 +2,7 @@ const CookieModel = require('../../../common/models/Cookie.class');
 const logger = require('../../../common/utils/logger')(__filename);
 const garApi = require('../../../common/services/garApi');
 const manifestFields = require('../../../common/seeddata/gar_manifest_fields.json');
+const airportValidation = require('../../../common/utils/airportValidation');
```

Comment:
Why did you need to add this in?

Reply:
From discussion, it was that opensearch raised airport validaiton error, get controller did not have an airport validaiton reference, thus I added it.
* Seemed to be trigger on the amend get.controller.